### ISSUE TITLE
How can I make a new Crick params file?

### ISSUE THREAD

**Issue by JasonIsaac-Lofty** [tfhuddy/2023-manuscript-materials#2]
Hi，
This work is wonderful and gives me a lot of inspiration! 
I want to create a new Crick params file which has a omega1 value equals to 98.2°. However in Rosetta database(main/database/protocol_data/crick_parameters) has only 2 preset crick params files with omega1 value of 98.65° and 100°. 
In this 2 files, there are slightly different in r1, z1, delta_omega1 and delta_z1. I don't know how to adjust them to fit omega1 value of 98.2°.
The comment line mentioned a app called **_fit_helixparams_**, but I can't find this app in public released Rosetta and in documentation of Rosettacommons website.
Is there a way to adjust these values manually? Any suggestions would help.
Thank you very much.

**Comment by tfhuddy** [comment]
Hi,

Thanks, and I am glad you are gonna do some cool stuff!. See if this first
supp methods section from Po-Ssu Huang's hyperstable bundle paper helps
(attached). Depending on how funky you want to make the helices, this may
or may not be sufficient to help you determine parameters to write a new
file to deposit in that crick_parameters directory (described here:
https://docs.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/Movers/Crick-params-files
)

Also if you already have a backbone shape like the one you desire, there
are also alternative methods to copy and displace the copies of it if you
want, which I can share some examples of.

Let me know how it goes!
Tim

**Comment by JasonIsaac-Lofty** [comment]
Hi,

Thank you for your reply! I tested present Crick params file "alpha_helix" by changing the subtag _helix_length_ in <MakeBundle> mover given in this repo.  I'm also studying the crick params file now! I don't know if I can change the Crick params file directly?

And I read Po-Ssu Huang's work, the method provided in this ariticle is very similar to another RosettaScript mover **_BundleGridSampler_**, I'm trying to use this mover to create the helix bundle desired.

All the best.

**Comment by tfhuddy** [comment]
Did you change that value without altering the start pose duplicate
residues in the repeat propagation mover? If you want these values to
always be tied together, you can set them to the same variable
"%%var_name%%" in the xml and then use cmd line option -parser:script_vars
var_name=11 or something like that. It sounds like the mismatch just made
the repeat propagation incorrect which will distort your pose most likely

**Comment by JasonIsaac-Lofty** [comment]
Yes! That's the problem! Now it works properly. Thank you very much!
